avalanche/training/plugins/lfl.py

```diff
@@ -0,0 +1,81 @@
+import copy
+
+import torch
+
+from avalanche.training.plugins.strategy_plugin import StrategyPlugin
+from avalanche.training.utils import get_last_fc_layer, freeze_everything
+from avalanche.models.base_model import BaseModel
+
+
+class LFLPlugin(StrategyPlugin):
+
+    def __init__(self, lambda_e):
```

Comment:
can you add more details here? or at least link to `LFL(BaseStrategy)`'s documentation.